internal/contour/listener_test.go

```diff
@@ -550,12 +621,21 @@ func TestListenerVisit(t *testing.T) {
 			root := reh.Build()
 			got := visitListeners(root, &tc.ListenerVisitorConfig)
 			if !reflect.DeepEqual(tc.want, got) {
```

Comment:
We probably don't need to use both reflect and cmp here. You can use !cmp.Equal(tc.want, got) here